db/config.cc

```diff
@@ -673,6 +673,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         " It is not enough to have ever since upgraded to newer versions of Cassandra. If you EVER used a version earlier than 2.1 in the cluster where these SSTables come from, DO NOT TURN ON THIS OPTION! You will corrupt your data. You have been warned.")
     , enable_shard_aware_drivers(this, "enable_shard_aware_drivers", value_status::Used, true, "Enable native transport drivers to use connection-per-shard for better performance")
     , enable_ipv6_dns_lookup(this, "enable_ipv6_lookup", value_status::Used, false, "Use IPv6 address resolution")
+    , abort_on_internal_error(this, "abort_on_internal_error", value_status::Used, false, "Abort the server instead of throwing exception when internal invariants are violated")
 
```

Comment:
Please make this LiveUpdate. Reasoning: we see errors, user is prepared for a core dump, enables setting, captures core dump, disables setting, ships core dump for analisys

Reply:
Ok.